libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -117,14 +117,14 @@ const AP_Param::GroupInfo AC_AutoTune_Heli::var_info[] = {
     // @Description: maximum angular acceleration in deg/s/s allowed during autotune maneuvers
     // @Range: 1 4000
     // @User: Standard
```

Comment:
We could add units markup to the param.
```suggestion
    // @User: Standard
    // @Units: deg/s/s
```